README.md

```diff
@@ -1,60 +1,92 @@
-# Source Code Annotation application
+[![Build Status](https://travis-ci.org/src-d/code-annotation.svg)](https://travis-ci.org/src-d/code-annotation)
+![unstable](https://svg-badge.appspot.com/badge/stability/unstable?a)
+
+# Source Code Annotation Tool
 
 In order to evaluate quality of ML models, as well as to create “ImageNet for source core” there is a need for tools to automate the data collection/labeling/annotation.
 
+![Screenshot](.github/screenshot.png?raw=true)
+
 ## Installation
 
 ### Github OAuth tokens
 
-First you need OAuth application on github. [Read how to create it](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
+1. You need OAuth application on github. [Read how to create it](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
+
+    `Authorization callback URL: http://127.0.0.1:8080/oauth-callback`
 
-On a [page](https://github.com/settings/developers) with your application you will need `Client ID` and `Client Secret`
+2. Copy `.env.tpl` to `.env`.
 
-Copy `.env.tpl` to `.env` and set tokens there.
+3. On a [page](https://github.com/settings/developers) with your application find `Client ID` and `Client Secret` and put them in `.env` file.
```

Comment:
3. Retrieve the values for your application's `Client ID` and `Client Secret` from the [GitHub Developer Settings page](...) and add them to the end of the corresponding lines in `.env`.